client/locations-list-map.js

```diff
@@ -231,10 +232,12 @@ function createFilters(data) {
     },
   };
 
+  // All states are included as potential filters through query and not selectable.
```

Comment:
"through query and not selectable." <-- this is slightly unclear to me. Are you saying

States are always included as potential filters and clients select it via the URL query parameters
even if they are not selectable?

Maybe?